components/PageHeader.vue

```diff
@@ -88,7 +88,9 @@
     },
     methods: {
       submitSearchForm () {
-        const newSearchQuery = { ...this.searchQuery, ...{ query: this.query } };
+        const view = this.$route.query.view || sessionStorage.searchResultsView || localStorage.searchResultsView || 'grid';
```

Comment:
I think we probably don't want this component to need be concerned with the source of the various parameters other than `query`. How about instead we add to layouts/default.vue:
```js
    mounted () {
      if (!this.searchQuery.hasOwnProperty('view')) {
        this.searchQuery.view = sessionStorage.searchResultsView || localStorage.searchResultsView || 'grid';
      }
    }
```